src/Compilers/VisualBasic/Portable/Emit/EditAndContinue/VisualBasicSymbolMatcher.vb

```diff
@@ -692,6 +692,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Emit
             End Function
 
             Public Overrides Function VisitNamedType(type As NamedTypeSymbol) As Symbol
+                If (type.IsTupleType) Then
```

Comment:
the original question was in [https://github.com/dotnet/roslyn/pull/19114](https://github.com/dotnet/roslyn/pull/19114)

re-reviewing it now

---
In reply to: [116339869](https://github.com/dotnet/roslyn/pull/19283#discussion_r116339869) [](ancestors = 116339869)